notebooks/create-validation-dataset.ipynb

```diff
@@ -0,0 +1,225 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    validation_set
</pre><p> </p><p>Do we want to store the dataset somewhere?  Or maybe even just have the notebook write it to a file</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>+1 </p><p>Shall we save it somewhere in the data folder? We will be using it in future for evaluating our generate QA model outputs. </p>